core/modules/update/update.theme.inc

```diff
@@ -42,8 +42,9 @@ function theme_update_last_check($variables) {
   $output = '<div class="update checked">';
 
   $output .= $last ? t('Last checked: @time ago', array('@time' => format_interval(REQUEST_TIME - $last))) : t('Last checked: never') . ' ';
-  $output .= l(t('Check manually'), 'admin/reports/updates/check', array('attributes' => array('class' => array('button', 'button-secondary')), 'query' => backdrop_get_destination()));
-
+  if (user_access('Administer site configuration')) {
```

Comment:
Sorry, I gave you bad advice again. The permission must be in lower case. Simply change "Adminster" to "administer".

`if (user_access('administer site configuration')) {`

Reply:
Can you just edit it? I will commit suggestion

Reply:
This is quite strange. I didn't think I could, but I tried and I was in fact able to.